nubilanding/src/components/retiros/styles.module.scss

```
@@ -0,0 +1,97 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,500|Rubik:400,500&display=swap'); 
+
+.contenedorGeneral {
+    display: flex; 
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    background-image: url('../../assets/bg_retiros.png');
+    background-repeat: no-repeat;
+    background-size: 150%;
+}
+
+.contenedorRetira {
+    width: 40%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.contenedorTabla {
+    width: 40%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-left: 5%;
+}
+
+.tabla ,td ,th{
```

Prestar atención a los espacios hace que el código se vea más prolijo:
```suggestion
.tabla ,td ,th {
```